docs/cmdline-opts/cert.d

```diff
@@ -14,19 +14,24 @@ Tells curl to use the specified client certificate file when getting a file
 with HTTPS, FTPS or another SSL-based protocol. The certificate must be in
 PKCS#12 format if using Secure Transport, or PEM format if using any other
 engine. If the optional password is not specified, it will be queried for on
-the terminal. Note that this option assumes a \&"certificate" file that is the
-private key and the client certificate concatenated! See --cert and --key to
-specify them independently.
+the terminal if required. Note that this option assumes a \&"certificate"
+file that is the private key and the client certificate concatenated! See
+--cert and --key to specify them independently.
 
 If curl is built against the NSS SSL library then this option can tell
 curl the nickname of the certificate to use within the NSS database defined
 by the environment variable SSL_DIR (or by default /etc/pki/nssdb). If the
 NSS PEM PKCS#11 module (libnsspem.so) is available then PEM files may be
-loaded. If you want to use a file from the current directory, please precede
-it with "./" prefix, in order to avoid confusion with a nickname. If the
-nickname contains ":", it needs to be preceded by "\\" so that it is not
-recognized as password delimiter. If the nickname contains "\\", it needs to
-be escaped as "\\\\" so that it is not recognized as an escape character.
+loaded.
+
+If you intend to use a certificate file and provide a path relative to the
+current directory, you must prefix the path with "./" in order to avoid
+confusion with an NSS database nickname.
+
+If the NSS nickname or certificate filename contains the character ":", it
+must be prefixed by "\\" so that it is not recognized as the password
+delimiter. Similarly, if the nickname or filename contains "\\", it must be
+escaped as "\\\\" so that it is not recognized as an escape character.
```

Comment:
You add the mention of the escaping for file names in the section for NSS details which not reads as if this is only for NSS somehow, when it really isn't. Right?

Reply:
It sounds sensible.